lib/matplotlib/collections.pyi

```diff
@@ -4,9 +4,10 @@ from typing import Literal
 import numpy as np
 from numpy.typing import ArrayLike, NDArray
 
-from . import artist, cm, transforms
+from . import colorizer, transforms
 from .backend_bases import MouseEvent
 from .artist import Artist
+from .colorizer import Colorizer
```

Comment:
This seems redundant to `colorizer` being imported above. Probably only need one or the other way.